scripts/jenkins/ardana/ansible/bootstrap-crowbar-nodes.yml

```diff
@@ -80,8 +46,21 @@
         - name: Gather facts
           setup:
 
+        - name: Disable DHCP on the cloud infra management network
+          command: |
+            openstack --os-cloud {{ os_cloud }} subnet set \
+              --no-dhcp \
+              openstack-ardana-{{ ardana_env }}_management_subnet
+          delegate_to: localhost
+          run_once: True
+
         - include_role:
             name: setup_root_partition
+
+        # FIXME: remove everything but eth0 ifcfg files from base image
+        - name: Remove unused ifcfg files
```

Comment:
why do we need that? some explanation in the comment would be good for the future us.

Reply:
Added a note explaining why